indico/web/client/js/utils/redux.js

```diff
@@ -29,7 +33,7 @@ function _combineReducers(...reducers) {
                 // it's a reducer
                 state = reducer(state, action);
             } else {
-                state = combineReducers(reducer)(state, action);
+                state = memoizedCombineReducers(reducer)(state, action);
```

Comment:
Wouldn't it be cleaner to do the combining once when `_combineReducers()` is called instead of when the `(state, action) => ...` function gets called?

Reply:
Yeah, I thought of that but it complicated the code quite a bit. But I believe I have another idea.